ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -22,6 +23,7 @@ def setUp(self):
         super().setUp()
         self.order_page = Order()
         self.orders_api = OrdersAPI()
+        self.my_profile_page = MyProfile()
```

Comment:
move it to the related test cases as almost all other test cases won't use it

Reply:
Done